utils.js

```diff
@@ -4266,14 +4479,39 @@ export function repoMetadataToGitHubApiUrl(repoMetadata) {
     const name = repoMetadata.name;
     let ghUrl = "https://api.github.com/repos";
     if (group && group !== "." && group !== "") {
-      ghUrl = `${ghUrl}/${group.replace("github.com/", "")}`;
+      ghUrl = `$ghUrl/${group.replace("github.com/", "")}`;
     }
     ghUrl = `${ghUrl}/${name}`;
     return ghUrl;
   }
   return undefined;
 }
 
+/**
+ * Method to run cli command `pixi install`
+ *
+ *
+ */
+export function generatePixiLockFile(path) {
+  const result = spawnSync("pixi", ["install"]);
```

Comment:
Use the spawnSync snippet from another method. For Windows, we need shell, encoding to utf-8 etc.